flow/connectors/postgres/client.go

```diff
@@ -529,6 +530,7 @@ func (c *PostgresConnector) generateMergeStatement(destinationTableIdentifier st
 	for i, columnName := range columnNames {
 		columnNames[i] = fmt.Sprintf("\"%s\"", columnName)
 	}
+	parsedDstTable, _ := utils.ParseSchemaTable(destinationTableIdentifier)
```

Comment:
On one level it feels like this could be made more efficient with a `utils.SanitizeSchemaTable` where it'd avoid allocations in case where `destinationTableIdentifier == parsedDstTable.String()` but in another level it seems like you're already too late here if for stranger names like those containing a period. But maybe character stripping avoids that for now